package.json

```diff
@@ -39,27 +39,27 @@
   },
   "dependencies": {
     "@authenio/samlify-node-xmllint": "^2.0.0",
-    "@aws-sdk/client-route-53": "^3.750.0",
+    "@aws-sdk/client-route-53": "^3.758.0",
     "@chakra-ui/icons": "^2.2.4",
-    "@chakra-ui/react": "^2.7.1",
+    "@chakra-ui/react": "^2.10.6",
     "@emotion/react": "^11.14.0",
     "@emotion/server": "^11.11.0",
     "@emotion/styled": "^11.14.0",
-    "@prisma/client": "^6.4.0",
-    "@remix-run/express": "2.15.3",
-    "@remix-run/node": "2.15.3",
-    "@remix-run/react": "2.15.3",
-    "@remix-run/server-runtime": "2.15.3",
+    "@prisma/client": "^6.4.1",
+    "@remix-run/express": "2.16.0",
+    "@remix-run/node": "2.16.0",
+    "@remix-run/react": "2.16.0",
+    "@remix-run/server-runtime": "2.16.0",
     "acme-client": "^5.4.0",
     "bad-words": "^3.0.4",
-    "bullmq": "^5.41.3",
+    "bullmq": "^5.41.7",
     "compression": "^1.8.0",
     "cors": "^2.8.5",
     "cross-env": "^7.0.3",
     "dayjs": "^1.11.13",
     "docker-secret": "^2.0.6",
-    "express": "^4.21.2",
-    "framer-motion": "^12.4.5",
+    "express": "^5.0.1",
```

Comment:
Express 5 is tagged `next` on npmjs.com, indicating it's not yet stable:

https://www.npmjs.com/package/express?activeTab=versions

![image](https://github.com/user-attachments/assets/c9c94201-d66f-43f7-9bb6-c71f015aaeb6)

Reply:
Agree, let's not move to Express 5 yet